adafruit_is31fl3731/matrix.py

```diff
@@ -61,19 +69,17 @@ def image(self, img, blink=None, frame=None):
             raise ValueError("Image must be in mode L.")
         if img.size[0] != self.width or img.size[1] != self.height:
             raise ValueError(
-                "Image must be same dimensions as display ({0}x{1}).".format(
-                    self.width, self.height
-                )
+                f"Image must be same dimensions as display {self.width}x{self.height}"
             )
 
         # Frame-select and then write pixel data in one big operation
-        if frame is not None:
+        if frame != 0:
```

Comment:
The rest of these were left checking for None. I think we should keep them consistent if possible so this one can change back to check for None or the others could change to check for 0 but I think that may slightly change the behavior in some cases so I lean towards reverting back to just using `None` instead of 0.